### DISCUSSION TITLE
down/

### DISCUSSION THREAD

**Discussion by giscus** [Capchdo/blog#3] [General]
# down/

关于 HaoBIT 等。

https://blog.haobit.top/down/

**Comment by CZLeader** [comment]
评论区失而复得

https://github.com/Capchdo/blog/pull/21

**Reply by YDX-2147483647** [reply]
有了自动重启后，不太容易宕机了；~~你制造了最近的最大事故。~~

**Reply by CZLeader** [reply]
内存泄漏怎么办？
只有殺！
